### PR TITLE
chore: add vscode config file to include eslint working directories

### PR DESCRIPTION
**Pull request by sauldeleon**
Include VSCode configuration file based on Readme configuration. Based on 
https://github.com/belgattitude/nextjs-monorepo-example/pull/737#issuecomment-960242070

Let me check if this helps!

Thanks for your awesome monorepo!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -48,7 +48,7 @@ yarn-error.log*
 *.launch
 *.sublime-workspace
 
-.vscode/
```
Comment (sauldeleon) [inline .gitignore:51]: With this rule, no matter what we include in the vscode folder and which rules we set, no file will be added to the stage

**Comment by belgattitude** [comment]
Hey @sauldeleon thanks a lot...

I'm not a vscode user, so I generally 

- try not add code that I'm not able to maintain (would you be available for help ? :smile:  
- and I feel that vscode might be harder to do.

I need some time to chew on this, but definitely I'll come back in few days. 

Because I feel it's a great improvement

**Comment by sauldeleon** [comment]
Well, I understand that adding editor configuration into a monorepo may be something unusual in a project.

But I see this as a boilerplate code that people can fork to start their projects, and when using VScode some eslint stuff is failing the first impression is: This is not working properly, when the real situation is that is working perfectly but needs a bit more tweaking.

The normal approach is to read the Readme in which is documented what to do with VSCode, but... xD

I would say let me play arround a bit more with the monorepo in my fork, as there are still some more issues when using VScode and will update this PR. Or close it and I will open a new one.

**Comment by belgattitude** [comment]
>  I understand that adding editor configuration into a monorepo may be something unusual in a project.

It's really welcome and appreciated, will help a lot of people I'm sure.

> I would say let me play around a bit more with the monorepo in my fork, as there are still some more issues when using VScode and will update this PR.

Thanks millions, yes that would be awesome :heart:  

To give a bit the context

- doc was added here: https://github.com/belgattitude/nextjs-monorepo-example/pull/737
- interesting comment: https://github.com/belgattitude/nextjs-monorepo-example/issues/687#issuecomment-965554144
- and vscode discussion: https://github.com/microsoft/vscode/issues/40233

**Comment by belgattitude** [comment]
Hey @sauldeleon I'll close this one. Feel free to re-open.

Just found out this would relatively do the job:  https://github.com/belgattitude/nextjs-monorepo-example/pull/1848. But I'm still not using vscode on everyday basis. Let me know if it helps

**Pull request closed without merging**